phpunit_autoloader.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Are you sure you need a custom bootstrapping/autoloader, and what do these methods do, if you don't mind explaining? See my other comments on the test namespace.

Reply:
The custom bootstrapper/autoloader is needed in order to run the Unit tests in the SDK project itself. Since the SDK isn't loaded within itself in composer the composer autoloader has no knowledge of it.

If there is something I'm missing we can 100% change this but it's also internal only so less of a priority unless you know how to make it work without spending much time.